src/lib.rs

```diff
@@ -30,4 +30,4 @@ pub use verso::Verso;
 /// Re-exporting Winit for the sake of convenience.
 pub use winit;
 /// Context
-pub mod context_menu;
+pub mod components;
```

Comment:
We can reuse `webivew` module I think and move context_menu, panel, and prompt under there.

Reply:
renamed to webview and moved WebView Panel into it